package.json

```diff
@@ -55,7 +56,10 @@
     "axios": "^0.27.2",
     "bignumber.js": "^9.0.1",
     "eth-rpc-errors": "^4.0.3",
-    "ethers": "^5.6.6"
+    "ethers": "^5.6.6",
+    "i": "^0.3.7",
+    "npm": "^8.12.1",
+    "qs": "^6.10.5"
```

Comment:
Are we sure we want these as dependencies?

Reply:
Nope, I needed these for changes I ended up removing.